src/fe/implicit/_.js

```diff
@@ -62,54 +124,88 @@ export const now = Date.now;
 export const random = Math.random;
 export const floor = Math.floor;
 
-// function utils
+/**
+ * Calls a function after a given time and returns a function to clear the timeout
+ * @param {function} func
+ * @param {number} delay
+ *
+ * @returns {function}
+ */
 export const timeout = (func, delay) => {
   var timerId = setTimeout(func, delay || 0);
   return () => clearTimeout(timerId);
 };
 
+/**
+ * Calls a function at a set interval of time and returns a function to clear the interval.
+ * @param {function} func
+ * @param {number} delay
+ *
+ * @returns {function}
+ */
 export const interval = (func, delay) => {
   var timerId = setInterval(func, delay || 0);
   return () => clearInterval(timerId);
 };
 
-// returns a function which tells elapsed time at that point
-// on each subsequent invocation
+// remaining
```

Comment:
```suggestion
/**
 * Returns a function that returns the time
 * that has passed since this function was
 * invoked.
 * 
 * @returns {function (): number}
 */
```